hri/packages/speech/scripts/useful_audio.py

```diff
@@ -123,6 +123,99 @@ def debug(self, message):
         if self.debug_mode:
             self.log(message)
 
+    def build_audio(self, data):
+        if self.voiced_frames is None:
+            self.voiced_frames = data
+        else:
+            self.voiced_frames += data
+        self.chunk_count += 1
+
+    def discard_audio(self):
+        if not self.service_active:
+            self.ring_buffer.clear()
+            self.voiced_frames = None
+            self.chunk_count = 0
+
+    def publish_audio(self):
+        if self.chunk_count > MIN_CHUNKS_AUDIO_LENGTH:
+            self.publisher.publish(AudioData(data=self.voiced_frames))
+        self.discard_audio()
+
+    def int2float(self, sound):
+        abs_max = np.abs(sound).max()
+        sound = sound.astype("float32")
+        if abs_max > 0:
+            sound *= 1 / 32768
```

Comment:
can we have some constants managment?? I see a lot of numerical arbitrary numbers, maybe attach them with the semantic values of them